xml/FinanceType.xml

```diff
@@ -810,10 +602,6 @@
                 <narrative>Purchase of securities from issuing agencies</narrative>
                 <narrative xml:lang="fr">Titres et autres instruments émis par les agences multilatérales</narrative>
             </name>
-            <description>
```

Comment:
@akmiller01 can you see a reason why this was removed?

Reply:
There was code to remove descriptions where narratives are empty; would we rather not do that?

Reply:
Ok, that makes logical sense. I just did not understand why they were removed.

Reply:
Seems fine to remove this stuff, but I wonder if it should be a separate PR. It creates a lot of noise in this one, which also contains important changes.

Reply:
Agreed. Reversed for now.

Reply:
Ah – that’s much better! Thanks